tests/v2/validation/charts/monitoring.go

```diff
@@ -183,31 +184,37 @@ func checkPrometheusTargets(client *rancher.Client) (bool, error) {
 
 // editAlertReceiver is a private helper function
 // that edits alert config structure to be used by the webhook receiver.
-func editAlertReceiver(alertConfigByte []byte, origin string, originURL *url.URL) ([]byte, error) {
-	alertConfig, err := config.Load(string(alertConfigByte))
+func editAlertReceiver(alertConfigByte []byte, originURL *url.URL) ([]byte, error) {
+	alertConfig := &resources.AlertmanagerConfig{}
+	err := yaml.Unmarshal(alertConfigByte, alertConfig)
 	if err != nil {
 		return nil, errors.Wrapf(err, "failed to unmarshal alert config")
 	}
 
-	alertConfig.Global = &config.GlobalConfig{
+	vsendresolved := false
+
+	alertConfig.Global = &resources.GlobalConfig{
 		ResolveTimeout: alertConfig.Global.ResolveTimeout,
 	}
-	alertConfig.Receivers = append(alertConfig.Receivers, &config.Receiver{
+	alertConfig.Receivers = append(alertConfig.Receivers, &resources.Receiver{
 		Name: webhookReceiverDeploymentName,
-		WebhookConfigs: []*config.WebhookConfig{
+		WebhookConfigs: []*resources.WebhookConfig{
 			{
-				HTTPConfig: &config.HTTPClientConfig{
-					ProxyURL: config.URL{URL: originURL},
-				},
-				NotifierConfig: config.NotifierConfig{
-					VSendResolved: false,
+				VSendResolved: &vsendresolved,
+				HTTPConfig: &resources.HttpClientConfig{
+					ProxyURL: originURL.String(),
 				},
-				URL: origin,
+				URL: originURL.String(),
 			},
 		},
 	})
 
-	byteAlertConfig := []byte(alertConfig.String())
+	//byteAlertConfig := []byte(alertConfig.String())
```

Comment:
Just a small nit, can we remove this? I think this is from the dev process. Just to prevent confusion later on